qemu/tests/cfg/balloon_stress.cfg

```diff
@@ -1,24 +1,26 @@
 - balloon_stress:
     type = balloon_stress
+    virt_test_type = qemu
     only Windows
     kill_vm = yes
     repeat_times = 10000
-    virt_test_type = qemu
+    sub_test = win_video_play
     extra_params += " -balloon virtio"
-    video_url = http://FILESHARE.COM/pub/section2/kvmauto/video/big_buck_bunny_480p_stereo.avi
+    video_url = 
     x86_64:
         program_files = "%ProgramFiles(x86)%"
     i386:
         program_files = "%ProgramFiles%"
-    Windows:
-        session_cmd_timeout = 240
-        # disable first startup guide for windows media player
-        pre_cmd = "reg add HKLM\SOFTWARE\Policies\Microsoft\WindowsMediaPlayer /v GroupPrivacyAcceptance  /t REG_DWORD /f /d 00000001"
-        # enable repeat mode of windows media player
-        pre_cmd += " & reg add HKCU\Software\Microsoft\MediaPlayer\Preferences /v ModeLoop /t REG_DWORD /d 1 /f"
-        play_video_cmd  = taskkill /IM wmplayer.exe /F & "${program_files}\Windows Media Player\wmplayer.exe"  "${video_url}"  /play /fullscreen
-        check_playing_cmd = "tasklist|findstr /I wmplayer"
-        driver_name = "balloon.sys"
-        stop_player_cmd = "taskkill /IM wmplayer.exe /F"
-        # reset media player play mode
-        post_cmd += "reg add HKCU\Software\Microsoft\MediaPlayer\Preferences /v ModeLoop /t REG_DWORD /d 0 /f"
+    check_playing_cmd = "tasklist|findstr /I mplayer"
+    driver_name = "balloon.sys"
+    #Disable first startup guide for windows media player
+    wmplayer_reg_cmd = "reg add HKLM\SOFTWARE\Policies\Microsoft\WindowsMediaPlayer /v GroupPrivacyAcceptance  /t REG_DWORD /f /d 00000001"
```

Comment:
why still need above command, you can add execute above command in your setup/install.exe